services/dynamic-sidecar/Dockerfile

```diff
@@ -105,6 +106,10 @@ FROM base as production
 ENV SC_BUILD_TARGET=production \
   SC_BOOT_MODE=production
 
+RUN apt-get update &&\
+  apt-get install -y --no-install-recommends \
+  libmagic1
```

Comment:
why don't you actually install libmagic1 in the base image since it's used everywhere?

Reply:
I had to do it like this in the webserver because it did not work. Can give it a try.

Reply:
Apparently this works!